script.js

```diff
@@ -1 +1,19 @@
+const todoInput = document.getElementById("todo-input");
+const addTodoBtn = document.getElementById("add-todo-btn");
+const taskListContainer = document.getElementById("task-list-container");
+const taskList = document.getElementById("task-list");
 
+addTodoBtn.addEventListener("click", addTodo);
```

Comment:
The function names of event listeners should be prefixed with `on` or `handle` follwing the conventions.

```suggestion
addTodoBtn.addEventListener("click", onAddTodo);
```